runtime/doc/vim_diff.txt

```diff
@@ -437,6 +437,9 @@ Working directory (Vim implemented some of these later than Nvim):
 - `getcwd(-1)` is equivalent to `getcwd(-1, 0)` instead of returning the global
   working directory. Use `getcwd(-1, -1)` to get the global working directory.
 
+Keymap:
+- |nore| isn't applicable to <Plug> keymaps. <Plug> keymaps are always remaped.
```

Comment:
Personally I prefer "mappings" to "keymaps", especially in `vim_diff.txt`, as in Vim's terminology "mapping" and "keymap" are two things.
```suggestion
Mappings:
- |nore| is ignored for rhs <Plug> mappings. <Plug> mappings are always remapped.
```
BTW, paragraphs in this section are sorted alphabetically by their first line.

Reply:
> I feel this should also be documented in vim_diff.txt.

Sorry ? Paragraphs are sorted based on first line ? I don't I understood that part.

Reply:
That part is not important. You can ignore that.

Reply:
Done.